newsfragments/42343.significant.rst

```diff
@@ -0,0 +1,7 @@
+``DatasetManager.create_datasets`` now takes ``Dataset`` objects
```

Comment:
Same here, my understanding is `DatasetManager` is not part of the public interface, therefore, it is not needed to callout breaking changes?